haaska.py

```diff
@@ -527,7 +574,7 @@ def set_temperature(self, val, mode=None, state=None):
         self._call_service('climate/set_temperature', data)
 
 
-def mk_entity(ha, entity_id):
+def mk_entity(ha, entity_id, supported_features):
```

Comment:
Perhaps `supported_features` should be an optional arg if few callers need it, or there should be a wrapper for `mk_entity` that takes `payload` directly, to avoid all the duplicated calls to `payload_to_entity`.